docs/installation.md

```diff
@@ -148,6 +148,17 @@ server {
     listen 80 default_server;
     listen [::]:80 default_server;
 
+    gzip on;
+
+    gzip_vary on;
+    gzip_proxied any;
+    gzip_proxied expired no-cache no-store private auth;
+    gzip_comp_level 4;
+    gzip_buffers 16 8k;
+    gzip_http_version 1.1;
+    gzip_types text/plain text/css text/xml text/javascript application/x-javascript application/json application/xml $
```

Comment:
the $ character at the end, normally indicates a truncated string in nano and in this config no $ at the end should be necessary. please check again if this...

Reply:
Ah! Sorry!  Thank you for pointing that out... completely missed it... commits updated!